resources/nats-streaming/templates/istio-label-injection.yaml

```diff
@@ -0,0 +1,28 @@
+# This hook depends on helm creating the target namespace if it doesn't exist
+# before the hook is called. This is the case on Helm v2.9.1
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: label-ns
+  namespace: kube-system
```

Comment:
Why in the kube-system? and why with tiller SA which has high privillages ? and after https://github.com/kyma-project/kyma/issues/2073 you won't need to label the namespace with "enabled" value to have sidecars injected

Reply:
Agreed. Can we block this PR until next week? It would greatly simplify the process.

@piotrmsc I think he just copied what already exists in `kyma-system` namespace.

Reply:
@sjanota Yes, it was copied, tell me what should it be like, please. We can't block the PR, but we may delay the istio
@piotrmsc Can you delete it as a part of your #2073 ? to unlock the PR for now?

Reply:
@piotrmsc @sjanota I can create a follow-up ticket for us to remove it if you will not as part of your PR, it's only because it's a part of the AC in this ticket